Tests/TuistGeneratorTests/ProjectMappers/AutogeneratedSchemesProjectMapperTests.swift

```diff
@@ -1,4 +1,4 @@
-import Foundation
+import ProjectDescription
```

Comment:
For sure the prefixes are a bummer, but I had to bring this in for `TemplateString` for the test setup I borrowed from `ProjectNameAndOrganizationMapperTests`

Reply:
Ah that explains it, if required a regular `String` can be used instead of `TemplateString` for `TuistCore.Config` - that said, in this case `Config` is no longer required for this test as the subject `AutogeneratedSchemesProjectMapper` takes in a boolean :)